### ISSUE TITLE
setup.py?

### ISSUE THREAD

**Issue by Hierakonpolis** [MIC-DKFZ/HD-BET#62]
The readme reports the very reasonable instruction:
`Go into the repository (the folder with the setup.py file) and install`
However, at the moment I do not see any such file anywhere in the repository, and as expected `pip3 install -e` fails. What am I missing here?

**Comment by DininduSenanayake** [comment]
@Hierakonpolis Yeah, looks like the wording is incorrect. However,  `pip install -e .` should still work as long as you are on the parent directory **HD-BET** which has the pyproject.toml (not `HD_BET`) .

**Comment by DISAPPEARED13** [comment]
It seems that we don't have to run 'cd HD-BET', just install it in the parent directory is fine.

**Comment by Hierakonpolis** [comment]
Yep, that seems to work. Thanks!

**Comment by FabianIsensee** [comment]
Thanks for pointing this out. It's fixed in the documentation.
We had a new release on Friday. Might be worth checking out ;-)
